tendrl-common.spec

```diff
@@ -0,0 +1,57 @@
+Name: tendrl-common
+Version: 0.0.1
+Release: 1%{?dist}
+BuildArch: noarch
+Summary: Common Module for all Bridges
+Source0: %{name}-%{version}.tar.gz
+License: LGPLv2+
+URL: https://github.com/Tendrl/bridge_common
+
+BuildRequires: pytest
+BuildRequires: python2-devel
+BuildRequires: python-mock
+BuildRequires: python-six
+BuildRequires: systemd
+BuildRequires: python-yaml
+
+Requires: python-dateutil
+Requires: python-dns
+Requires: python-etcd
+Requires: systemd-python
+Requires: python-urllib3
```

Comment:
`Requires: python-six` as well, I think? `tendrl/common/definitions/validator.py` imports it